hawkpost/templates/account/auth_modals.html

```diff
@@ -98,15 +88,27 @@ <h3 class="title"> Sign up</h3>
             </label>
           </div>
           {% if redirect_field_value %}
-            <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
+          <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
           {% endif %}
         </div>
         <div class="col-xs-12">
-          <button class="btn-default" type="submit">{% trans "Sign Up" %}</button>
+          <button class="btn-default mt-small" type="submit">{% trans "Sign Up" %}</button>
+        </div>
+        <div class="col-xs-12 middle-xs flex-in">
+          <p class="no-padding">Sign up using:</p>
+          <ul class="no-padding">
+            {% include "socialaccount/snippets/provider_list.html" with process="login" %}
+          </ul>
+        </div>
+        <div class="col-xs-12">
+          <p class="smalltext">{% blocktrans %}Already have an account? Then please <a class=" md-trigger text-darkest" href="#modal-login" data-modal="modal-login">Sign in</a>{% endblocktrans %}</p>
         </div>
-        <p>{% blocktrans %}Already have an account? Then please <a class=" md-trigger" href="#modal-login" data-modal="modal-login">Sign in</a>.{% endblocktrans %}</p>
       </form>
     </div>
   </div>
 </div>
 <div class="md-overlay"></div>
+
+{% block scripts %}
```

Comment:
Since this template is included, I don't think this block is really used (or as any effect), because block are evaluated before includes (see the green block at the end of the chapter: https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#include). Please review if this file is really being loaded.